modules/account-lib/src/coin/sol/keyPair.ts

```diff
@@ -0,0 +1,34 @@
+import { Ed25519KeyPair } from '../baseCoin';
+import { NotImplementedError } from '../baseCoin/errors';
+import { DefaultKeys, KeyPairOptions } from '../baseCoin/iface';
+
+export class KeyPair extends Ed25519KeyPair {
+  /**
+   * Public constructor. By default, creates a key pair with a random master seed.
+   *
+   * @param { KeyPairOptions } source Either a master seed, a private key, or a public key
+   */
+  constructor(source?: KeyPairOptions) {
+    super(source);
+  }
+
```

Comment:
createKeyPair method left

Reply:
Keypair creation is done by the constructor